share/spice/dogo_news/dogo_news.js

```diff
@@ -0,0 +1,50 @@
+(function(env) {
+    "use strict";
+
+    env.ddg_spice_dogo_news = function(api_result) {
+        if (!api_result || !api_result.results || !api_result.results.length) {
+            return Spice.failed('dogo_news');
+        }
+
+        // Get original query.
+        var script = $('[src*="/js/spice/dogo_news/"]')[0],
+            source = $(script).attr("src"),
+            query = decodeURIComponent(source.match(/dogo_news\/([^\/]+)/)[1]);
+    
+        Spice.add({
```

Comment:
@dogomedia are you sure you wrapped everything in the `DDG.require` at the top? That should be all that's required to import Moment.js